pep/app/pods/user/model.ts

```diff
@@ -1,21 +1,46 @@
 import DS from 'ember-data';
 import attr from 'ember-data/attr';
+import moment, { Moment } from 'moment';
+import { SERVER_DATE_FORMAT } from 'pep/constants/dates';
 
 import { UserPreferences, DEFAULT_USER_PREFERENCES } from 'pep/constants/preferences';
 export enum UserType {
     GROUP = 'Group',
     INDIVIDUAL = 'Individual'
 }
 
+interface ActiveSubscription {
+    ProductCode: string;
+    ProductName: string;
+    OrderedViaName: string;
+    RecurringSubscriptionEndDate: Moment;
+    RecurringSubscriptionStartDate: Moment;
+}
+
 export default class User extends DS.Model {
+    @attr('string') activeSubscriptions!: string;
     @attr('boolean') branding!: boolean;
     @attr('string') brandingImgUrl!: string;
     @attr<any>('json', { defaultValue: () => ({ ...DEFAULT_USER_PREFERENCES }) }) clientSettings!: UserPreferences;
+    @attr('string') emailAddress!: string;
     @attr('boolean') hasArchiveAccess!: boolean;
     @attr('boolean') hasCurrentAccess!: boolean;
+    @attr('string') loggedInMethod!: string;
+    @attr('string') paDSHomeURL!: string;
     @attr('date') subscriptionEndDate!: Date;
+    @attr('string') userFullName!: string;
     @attr('string') userName!: string;
     @attr('string') userType!: UserType;
+
+    get activeSubscriptionsJSON(): ActiveSubscription[] {
+        return JSON.parse(this.activeSubscriptions).map((subscription: ActiveSubscription) => {
```

Comment:
Is it possible to have no active subscriptions? If so, we should check if an empty array or null gets returned and guard for it

Reply:
Yeah this is really weird because it's a string JSON array being returned. I'll add a check to see if it's an empty string/null and default to an empty array either way.

Reply:
Sounds good